plans/dns_jenkinsio.tf

```diff
@@ -43,12 +43,14 @@ locals {
 
   jenkinsio_cname_records = {
     # Azure
-    accounts = "public.aks.jenkins.io"
-    javadoc  = "public.aks.jenkins.io"
-    plugins  = "public.aks.jenkins.io"
-    reports  = "public.aks.jenkins.io"
-    www      = "jenkins.io"
-    uplink   = "public.aks.jenkins.io"
+    accounts       = "public.aks.jenkins.io"
+    archives.azure = "public.aks.jenkins.io"
```

Comment:
does this need .azure in it?

there's a dns record for archives.jenkins.io but it doesn't go anywhere just a blank apache page

Reply:
Right now there is only [http://archives.jenkins-ci.org](http://archives.jenkins-ci.org/ )listening on http only

That machine is running on rackspace and cost us around 500$/month, I would like to run archives.azure.jenkins.io during a testing period before switching officially to the k8s version 

Archives.jenkins-ci.org is managed by puppet with this [role](https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/manifests/archives.pp)